src/style.css

```diff
@@ -0,0 +1,50 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+body {
+  padding-top: 12px;
+}
+
+.search-input {
+  width: 100%;
+  height: 70px;
+  margin-bottom: 20px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: url(./images/Search.svg)  no-repeat 26px 25px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+}
+
+.search-input--small {
+  height: 42px;
+  background-position: 13px 15px;
+  background-size: 11px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-input:focus {
+  font-weight: bold;
+  border-radius: 3px;
+  background: url(./images/Search.svg)  no-repeat 26px 25px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+}
+
+.search-input--small:focus {
+  border-radius: 3px;
+  background: url(./images/Search.svg)  no-repeat 13px 15px / 11px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
you dont need to repeat all background styles cause you have defined it by default, just change sizes

Reply:
fix this